### DISCUSSION TITLE
Input Category Parameters in the Create Part screen

### DISCUSSION THREAD

**Discussion by angelosne** [inventree/InvenTree#7823] [Q&A]
Hello,
New user here.
I have created 2 part parameters (only admin can do that as I saw right?) footprint and smd/through.
Then in the settings->categories I add those to the category Resistors. 
Now when I create a part at the bottom it is enabled by default the option Copy Category Parameters and when I Submit the new part and go to the parameters I see the parameters I added and I can edit them.
Firstly is this the correct way to do that or is there any alternative.

I do not know if what I ask is possible or should be in the feature requests.
It would be more efficient to have the fields of those parameters in the Create Part screen and just select the values I want. With current workflow I have to edit each one of them

Thank you in advance

**Comment by SchrodingersGat** [comment]
@angelosne yes that is the current workflow. Implementing the parameters into the initial part creation screen is likely not going to happen
